app/lib/otis/daily_digest.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Used by DailyDigestMailer to report on action items requiring attention from
+# administrator.
+module Otis
+  class DailyDigest
+    attr_reader :ready_registrations, :expired_registrations,
+      :ready_approval_requests, :expired_approval_requests,
+      :expiring_users
+    def self.send
+      digest = DailyDigest.new
+      DailyDigestMailer.with(digest: digest).daily_digest_email.deliver_now
+    end
+
+    def initialize
+      @ready_registrations = HTRegistration.ready
+      @expired_registrations = HTRegistration.expired
+      @ready_approval_requests = HTApprovalRequest.approved.not_renewed
+      @expired_approval_requests = HTApprovalRequest.expired
+      @expiring_users = HTUser.active.select do |user|
```

Comment:
Probably fine given the relatively small number of users and that this isn't being run interactively, but I do wonder if it would be worth thinking about how to encode this as a SQL query so we don't have to load as many users.